channel.go

```diff
@@ -75,6 +82,10 @@ func (c *PaymentChannel) Send(ctx *Context, amount *BigInt) error {
 	})
 }
 
+func (c *PaymentChannel) Index() int {
```

Comment:
I suggest we keep the method name as `Idx`. When I wrote the comment I didn't remember that it was called `Idx` instead of `Index`... And do we actually need to call it `GetIdx` to be more Java-idiomatic?

Reply:
Yes.